### ISSUE TITLE
`@stylistic/linebreaks` never rewrites a line break standing in four of the raws a node holds

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#283]
Found while fixing #270, by asking which other texts of a node the fix walks past. Four more raws, and each is the same failure: the warning is reported, `--fix` reports a clean pass, the file comes back byte-identical, and a relint reports the same warning again, which is the failure mode of #126.

### What minimal example or steps are needed to reproduce the bug?

Seven stylesheets, each with one line break spelled `\n` while every other break of the file is spelled `\r\n`. The break that is spelled `\n` is shown as the line the stylesheet is broken over:

- `raws.afterName` of an at-rule — `@media` and `(min-width: 1px) { a { b: c; } }` on the line under it;
- `raws.between` of an at-rule — `@media (min-width: 1px)` and `{ a { b: c; } }` on the line under it;
- `raws.between` of a rule — `a` and `{ b: c; }` on the line under it;
- `raws.between` of a declaration — `a { b:` and `c; }` on the line under it;
- `raws.important` of a declaration — `a { b: c` and `!important; }` on the line under it;
- `raws.left` of a comment — `/*` and `a */` on the line under it, with `b { c: d; }` after it;
- `raws.right` of a comment — `/* a` and `*/` on the line under it, with `b { c: d; }` after it.

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/linebreaks": "windows"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json --fix
```

### What did you expect to happen?

Every one of the seven breaks rewritten, as a break standing in a selector, in a value, in a set of at-rule parameters or in the text of a comment already is.

### What actually happened?

Each is reported once — at `1:7`, `1:24`, `1:2`, `1:7`, `1:9`, `1:3` and `1:5` in the order above — `--fix` reports no warnings left, and each file is returned exactly as it went in. Linting the output reports the same position again.

### What is the cause?

The fix at [`lib/rules/linebreaks/index.js`](https://github.com/stylelint-stylistic/stylelint-stylistic/blob/main/lib/rules/linebreaks/index.js) walks the tree and rewrites a fixed list of texts: the selector, the value, the params, the text of a comment, `raws.before` and `raws.after` of every node, and `raws.after` of the root. None of the four above is on that list.

The warning, meanwhile, comes from a scan of `root.source.input.css` line by line, which sees every break the file holds, in whichever text of whichever node it stands. So the reading and the writing disagree about which breaks belong to the file — the same disagreement as #270, over the texts that issue left standing.

### Do you have a proposal to fix the bug?

Rewrite them where the walk already rewrites `raws.before` and `raws.after`. No custom syntax keeps a second copy of any of the four, so a bare write reaches each of them, and the guards they need are the ones those two already carry: `raws.afterName` and `raws.important` may be absent, and `raws.left` and `raws.right` belong to a comment alone.

`raws.between` of a declaration is the one to look at twice: it holds the colon along with the whitespace, so what is rewritten there is the whitespace around a character rather than whitespace on its own.
